eve/main.yml

```diff
@@ -20,6 +20,8 @@ models:
       OIDC_PASSWORD: 'end2end_password'
       OIDC_FIRST_NAME: 'hello'
       OIDC_LAST_NAME: 'world'
+      SHELL_UI_IMAGE: "registry.scality.com/playground/jbwatenberg/shell-ui:2.9-dev"
```

Comment:
Please find a proper place for this image. `playground` is a place for us to play with images, not for images to be actually commited in our code.

I can help in setting up the proper place if needed, feel free to ping me for that.

Reply:
cc: @JBWatenbergScality

Reply:
ticket created to take care of this issue -> [ZENKO-3405](https://scality.atlassian.net/browse/ZENKO-3405)

Reply:
We also have one in metalk8s : https://github.com/scality/metalk8s/issues/3269